shared/oval/sshd_use_approved_ciphers.xml

```diff
@@ -4,12 +4,12 @@
       <title>Use Only Approved Ciphers</title>
       <affected family="unix">
         <platform>multi_platform_rhel</platform>
+        <platform>multi_platform_debian</platform>
       </affected>
       <description>Limit the ciphers to those which are FIPS-approved and only
       use ciphers in counter (CTR) mode.</description>
     </metadata>
     <criteria operator="AND">
-      <extend_definition comment="Installed OS is certified" definition_ref="installed_OS_is_certified" />
```

Comment:
@openprivacy One thing to note is that, for Level 2, it's the combination of OpenSSL/GnuTLS and the kernel that get the FIPS certification. For Level 1, I think it's just the libraries themselves.

FISMA High systems need Level 2 if I remember correctly.

But, that means that you can readily upgrade to 7.3 and, if I've read the documentation correctly, you can upgrade the underlying components as long as the vendor has not made changes to the cryptographic functionality of the subsystem. @shawndwells would know more about this from the RHEL point of view, I just read the docs and try to implement the stuff.

However, you do have to put the kernel in fips mode with `fips=1` at boot time for 140-2 Level 2 to actually be met since that's how you get the additional internal validation.

Long story short, the Ubuntu client is nice, but they don't have the certs so you can't use it on systems processing sensitive information (see my comment above).